include/deal.II/base/tensor.h

```diff
@@ -2065,7 +2065,7 @@ invert (const Tensor<2,1,Number> &t)
 {
   Number return_tensor [1][1];
 
```

Comment:
Maybe adding an `using internal::NumberType<number>::value` would simplify reading the code below?

Reply:
Good idea. I'll give this a go once the other branch has been merged (since this one relies on that branch anyway).

Reply:
@luca-heltai I tried your suggestion inside the function declaration but it didn't work out:
> error: using declaration cannot refer to class member

I guess this is only valid for base class member functions.

I would be hesitant to add the alias `template<typename T> using value = internal::NumberType<T>::value` (or similar) outside of any function declaration, as it would then be permanently exposed outside of the internal namespace. It would be nice to somehow shorten each these lines. Do you have any other suggestions?

Reply:
Yes, the form of the `using` statement @luca-heltai proposed is only valid for (i) namespace names and (ii) base member names.